From d97a6a0499a51b638ae11bd359b9212e4be3ba5c Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Thu, 11 Oct 2018 19:04:09 +0700
Subject: [PATCH 699/909] Revert "arm64: dts: renesas: r8a7795-es1: Point
 DU/VSPD via FCPVD to IPMMU-VI0"

commit 75a3cd68737e59773aa1b00d5e33db60dfc6c440 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 6507eaf83cf2562da47e001dae34942df93fb941
to keep IPMMU address translation for FCDVD is disabled by default,
maintain the same IPMMU support environment with previous
released R-Car Linux BSP.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index a4b8a50c0f3a..f530775cd046 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -171,7 +171,6 @@
 		clocks = <&cpg CPG_MOD 600>;
 		power-domains = <&sysc R8A7795_PD_ALWAYS_ON>;
 		resets = <&cpg 600>;
-		iommus = <&ipmmu_vi0 11>;
 	};
 
 	fdp2: fdpm@fe948000 {
-- 
2.17.1

